regex2/challenges-11.test.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named validatePin that uses a regular expression pattern to validate a PIN.
+//
+// If the PIN is four numerical digits long, return true. Otherwise, return false.
+// ------------------------------------------------------------------------------------------------
+
+const validatePin = (pin) => {
+  return /^\d\d\d\d$/.test(pin);
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named findTagNames that iterates over an array of HTML strings
+// and uses a regular expression pattern to return the closing tags.
+//
+// For example, findTagNames(['<h1>Hello, world!</h1>', '<p>Welcome to my site</p>'])
+// returns ['/h1', '/p'], and findTagNames(['<div><h1>Hello, world!</h1></div>', '<p>Welcome to my site</p>'])
+// returns ['/h1', '/div', '/p'].
+// ------------------------------------------------------------------------------------------------
+
+const findTagNames = elements => {
+  let regex = /\/[a-z][a-z0-9]?[a-z]?/g;
```

Comment:
If I had added more tests... you'd need to make sure this would work for a `</header>` as well, and not for a `lose/lose`.